### ISSUE TITLE
build failed

### ISSUE THREAD

**Issue by kklepper** [oubiwann/docker-lfe-yaws-sample-app#1]
I'm on AWS EC2 ubuntu, no make, so I resolved to ``sudo docker build -t *** .`` I got 

    Compiled src/clj-comp.lfe
    ERROR: compile failed while processing /opt/sample-app/deps/clj: rebar_abort
    src/clj-seq.lfe:3: redefining function: #(LFE-EXPAND-USER-MACRO 3)
    Compiling /opt/sample-app/deps/clj/src/clj-seq.lfe failed:
    The command '/bin/sh -c git clone $APP_REPO $APP_DIR &&         cd $APP_DIR &&         ln -s $LFE_HOME $DEPS_DIR/lfe &&         rebar compile' returned a non-zero code: 1

Any idea?

**Comment by oubiwann** [comment]
Yeah, there are a whole new set of Docker images being built based on the official Erlang Docker images ... this include LFE and YAWS, and once that's finished, this issue will be sorted.
